src/qml/qgismobileapp.qml

```diff
@@ -810,24 +810,25 @@ ApplicationWindow {
                   digitizingFeature.geometry.applyRubberband()
                   digitizingFeature.applyGeometry()
                 }
-
                 if( !overlayFeatureFormDrawer.featureForm.featureCreated )
                 {
                     digitizingFeature.resetAttributes();
                     if( overlayFeatureFormDrawer.featureForm.model.constraintsHardValid ){
-                      //when the constrainst are fulfilled
-                      digitizingFeature.create()
-                      overlayFeatureFormDrawer.featureForm.featureCreated = true
+                      // when the constrainst are fulfilled
+                      // indirect action, no need to check for success and display a toast, the log is enough
+                      overlayFeatureFormDrawer.featureForm.featureCreated = digitizingFeature.create()
                     }
                 } else {
-                    digitizingFeature.save()
+                  // indirect action, no need to check for success and display a toast, the log is enough
+                  digitizingFeature.save()
                 }
             } else {
-                if( overlayFeatureFormDrawer.featureForm.featureCreated ) {
-                  //delete the feature when the geometry gets invalid again
-                  digitizingFeature.deleteFeature()
-                  overlayFeatureFormDrawer.featureForm.featureCreated = false
-                }
+              if( overlayFeatureFormDrawer.featureForm.featureCreated ) {
+                // delete the feature when the geometry gets invalid again
+                // indirect action, no need to check for success and display a toast, the log is enough
+                digitizingFeature.deleteFeature()
+                overlayFeatureFormDrawer.featureForm.featureCreated = false
```

Comment:
```suggestion
                overlayFeatureFormDrawer.featureForm.featureCreated = digitizingFeature.deleteFeature()
```
Not sure, but seems better to me.

Reply:
I think it should remain always `false`, as it is used a little bit further down to decide whether to call `digitizingFeature.create()` or `digitizingFeature.save()`. A deleteted feature should be saved, rather than created.
Check: 
https://github.com/opengisch/QField/pull/1018/files/c3038e3e592cd4b0005b19a7fd6b6c434515332b#diff-2692409a5f23502d8210c005af0e4787R859-R867

Even if I am wrong and it is used as sort of a success flag, we should change the name, because it is funny to assign on `featureCreated = deleteFeature()` :)

Reply:
Ah sorry. Of course it should remain true, if it cannot be deleted. 
So  `featureCreated = !deleteFeature()` or ` if (deleteFeature) featureCreated = false`.

Reply:
done!